



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<!-- ViewVC :: http://www.viewvc.org/ -->
<head>
<title>[pkgs] View of /devel/xsane/xsane-0.995-xdg-open.patch</title>
<meta name="generator" content="ViewVC 1.0.9" />
<link rel="stylesheet" href="/viewvc-static/styles.css" type="text/css" />

</head>
<body>
<div class="vc_navheader">

<form method="get" action="/viewvc/">

<table style="padding:0.1em;">
<tr>
<td>
<strong>

<a href="/viewvc/">

[pkgs]</a>
/

<a href="/viewvc/devel/">

devel</a>
/

<a href="/viewvc/devel/xsane/">

xsane</a>
/

<a href="/viewvc/devel/xsane/xsane-0.995-xdg-open.patch?view=log">

xsane-0.995-xdg-open.patch</a>


</strong>

</td>
<td style="text-align:right;">


<strong>Repository:</strong>
<select name="root" onchange="submit()">


<option value="*viewroots*">Repository Listing</option>










































































<optgroup label="CVS Repositories"><option>core</option><option>dirsec</option><option>docs</option><option>elvis</option><option>extras</option><option>fedora</option><option>l10n</option><option>legacy</option><option selected="selected">pkgs</option></optgroup>


</select>
<input type="submit" value="Go" />

</td>
</tr>
</table>

</form>

</div>
<div style="float: right; padding: 5px;"><a href="http://www.viewvc.org/"><img src="/viewvc-static/images/logo.png" alt="ViewVC logotype" width="128" height="48" /></a></div>
<h1>View of /devel/xsane/xsane-0.995-xdg-open.patch</h1>

<p style="margin:0;">

<a href="/viewvc/devel/xsane/"><img src="/viewvc-static/images/back_small.png" width="16" height="16" alt="Parent Directory" /> Parent Directory</a>

| <a href="/viewvc/devel/xsane/xsane-0.995-xdg-open.patch?view=log#rev1.1"><img src="/viewvc-static/images/log.png" width="16" height="16" alt="Revision Log" /> Revision Log</a>




</p>

<hr />
<div class="vc_summary">
Revision <strong>1.1</strong> -
(<a href="/viewvc/devel/xsane/xsane-0.995-xdg-open.patch?revision=1.1"><strong>download</strong></a>)

(<a href="/viewvc/devel/xsane/xsane-0.995-xdg-open.patch?annotate=1.1"><strong>annotate</strong></a>)

<br /><em>Fri Nov 23 11:03:41 2007 UTC</em>
(21 months, 2 weeks ago)
by <em>nphilipp</em>


<br />Branch: <strong>MAIN</strong>


<br />CVS Tags: <strong>xsane-0_996-2_fc11, xsane-0_996-6_fc11, xsane-0_997-1_fc12, xsane-0_996-4_fc11, xsane-0_995-1_fc9, xsane-0_996-7_fc12, F-10-split, xsane-0_996-9_fc12, xsane-0_996-8_fc12, F-11-split, xsane-0_996-10_fc12, F-9-split, xsane-0_996-1_fc11, xsane-0_995-3_fc9, xsane-0_995-2_fc9, xsane-0_995-4_fc10, xsane-0_997-2_fc12, xsane-0_996-5_fc11, xsane-0_997-3_fc12, xsane-0_995-5_fc10, xsane-0_996-3_fc11, HEAD</strong>








<pre class="vc_log">version 0.995
remove obsolete gimp2.0, medium-definitions, showeulaonce patches
</pre>

</div>
<div id="vc_markup"><pre>diff -up xsane-0.995/src/xsane.h.xdg-open xsane-0.995/src/xsane.h
--- xsane-0.995/src/xsane.h.xdg-open	2007-08-13 09:20:32.000000000 +0200
+++ xsane-0.995/src/xsane.h	2007-11-23 11:50:49.000000000 +0100
@@ -251,7 +251,7 @@
 # elif defined(HAVE_OS2_H)
 #  define DEFAULT_BROWSER "netscape"
 # else
-#  define DEFAULT_BROWSER "netscape"
+#  define DEFAULT_BROWSER "xdg-open"
 # endif
 #endif
 
</pre></div>

<hr />
<table>
<tr>
<td><address><a href="mailto:webmaster@fedoraproject.org">webmaster@fedoraproject.org</a></address></td>
<td style="text-align: right;"><strong><a href="/viewvc-static/help_rootview.html">ViewVC Help</a></strong></td>
</tr>
<tr>
<td>Powered by <a href="http://viewvc.tigris.org/">ViewVC 1.0.9</a></td>
<td style="text-align: right;">&nbsp;</td>
</tr>
</table>
</body>
</html>

